metashare/api/models.py

```diff
@@ -502,6 +504,7 @@ class ScratchOrg(PushMixin, HashIdMixin, TimestampsMixin, models.Model):
     config = JSONField(default=dict, encoder=DjangoJSONEncoder, blank=True)
     delete_queued_at = models.DateTimeField(null=True, blank=True)
     owner_sf_id = StringField(blank=True)
+    owner_username = StringField(blank=True)
```

Comment:
I wish this was clearer that this is a _github_ username, and that owner_sf_id is actually a Salesforce username rather than an Id.